<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterDialog</class>
 <widget class="QDialog" name="FilterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="lstInputFilter">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>111</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="lstDatabaseOutputFilter">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>111</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="lstGraphicInputFilter">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>111</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="AddInputFilter">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>350</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DelInputFilter">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>390</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Del</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DelDataVaseOutputFilter">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>390</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Del</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AddDataBaseOutputFilter">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>350</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DelGraphicInputFilter">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>390</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Del</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AddGraphicInputFilter">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>350</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>440</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QwtTextLabel" name="HWFilterText">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QwtTextLabel" name="DBFilterText">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QwtTextLabel" name="GraphicFilterText">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtTextLabel</class>
   <extends>QFrame</extends>
   <header>qwt_text_label.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
